main/src/main/java/org/mobilitydata/gtfsvalidator/validator/FeedServiceDateValidator.java

```diff
@@ -18,13 +18,16 @@
 
 import org.mobilitydata.gtfsvalidator.annotation.GtfsValidator;
 import org.mobilitydata.gtfsvalidator.annotation.Inject;
+import org.mobilitydata.gtfsvalidator.notice.MissingRequiredFieldError;
 import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
 import org.mobilitydata.gtfsvalidator.notice.StartAndEndDateOutOfOrderNotice;
 import org.mobilitydata.gtfsvalidator.table.GtfsFeedInfo;
 import org.mobilitydata.gtfsvalidator.table.GtfsFeedInfoTableContainer;
 
 /**
- * Validates that start_date <= end_date for all rows in "feed_info.txt".
+ * Validates : * start_date <= end_date for all rows in "feed_info.txt" * feed_info.start_date is
```

Comment:
If these are supposed to be bulleted it looks like the format isn't allowing this, and as a result it's hard to read. I'd use numbers instead of bullets or just write it out.

Reply:
Thanks.